assignments/002-unix-intro/unix-intro.sh

```diff
@@ -0,0 +1,670 @@
+DPCh_plate1_D06_S42.R2.fq.gz  DPCh_plate1_H06_S90.R2.fq.gz
+DPCh_plate1_D11_S47.R2.fq.gz  DPCh_plate1_H11_S95.R2.fq.gz
+DPCh_plate1_D12_S48.R2.fq.gz  DPCh_plate1_H12_S96.R2.fq.gz
+
+#### Ex. 12: Use the recursive (-r) option to remove `lonely-fastqs`
+####         and its contents
+
+rm -r lonely-fastqs/
+
+#### Ex. 13: Make two new directories in the top level of the
+####         repo called `fq_AB` and `fq_notAB`
+
+mkdir fq_AB/ fq_notAB/
+
+#### Ex. 14: Using the `[` `]` copy the R1 files in `fastqs` that
+####         belong to sample names (the part immediately after `DPCh_plate1_`)
+####         starting with an `A` or `B` to `fq_AB`.
+
+cp fastq/*_[AB]*.R1.* fq_AB
+
+#### Ex. 15: Now, using negation (`^`) with `[` and `]` copy all the
+####         R1 files that don't belong to sample names starting with an
+####         `A` or a `B` to `fq_notAB`. When you are done, on the next line in
+####         the code block, list the contents of `fq_AB` and `fq_notAB`.
+
+cp fastq/DPCh_plate1_[^AB]*R1* fq_notAB
+
+-----
+
+ls ls fq_notAB
+DPCh_plate1_C05_S29.R1.fq.gz  DPCh_plate1_F05_S65.R1.fq.gz
+DPCh_plate1_C06_S30.R1.fq.gz  DPCh_plate1_F06_S66.R1.fq.gz
+DPCh_plate1_C11_S35.R1.fq.gz  DPCh_plate1_F11_S71.R1.fq.gz
+DPCh_plate1_C12_S36.R1.fq.gz  DPCh_plate1_F12_S72.R1.fq.gz
+DPCh_plate1_D05_S41.R1.fq.gz  DPCh_plate1_G05_S77.R1.fq.gz
+DPCh_plate1_D06_S42.R1.fq.gz  DPCh_plate1_G06_S78.R1.fq.gz
+DPCh_plate1_D11_S47.R1.fq.gz  DPCh_plate1_G11_S83.R1.fq.gz
+DPCh_plate1_D12_S48.R1.fq.gz  DPCh_plate1_G12_S84.R1.fq.gz
+DPCh_plate1_E05_S53.R1.fq.gz  DPCh_plate1_H05_S89.R1.fq.gz
+DPCh_plate1_E06_S54.R1.fq.gz  DPCh_plate1_H06_S90.R1.fq.gz
+DPCh_plate1_E11_S59.R1.fq.gz  DPCh_plate1_H11_S95.R1.fq.gz
+DPCh_plate1_E12_S60.R1.fq.gz  DPCh_plate1_H12_S96.R1.fq.gz
+
+-----
+
+ls fq_AB
+DPCh_plate1_A05_S5.R1.fq.gz   DPCh_plate1_B05_S17.R1.fq.gz
+DPCh_plate1_A06_S6.R1.fq.gz   DPCh_plate1_B06_S18.R1.fq.gz
+DPCh_plate1_A11_S11.R1.fq.gz  DPCh_plate1_B11_S23.R1.fq.gz
+DPCh_plate1_A12_S12.R1.fq.gz  DPCh_plate1_B12_S24.R1.fq.gz
+
+#### Ex. 16: Remove `fq_AB` and `fq_notAB`, and their contents
+
+rm -r fq_AB/ fq_notAB/
+
+#### Ex. 17: Using the `{,}` construct create five directories named 
+####         `dir_A`, `dir_C`, `dir_E`, `dir_G`, and `dir_I`, and list them
+####         (but not their contents) when you are done. (Think `ls -d`).
+
+mkdir dir_{A,C,E,G,I}
+ls -d dir*
+
+-----
+
+dir_A  dir_C  dir_E  dir_G  dir_I
+
+#### Ex. 18: Using globbing, remove `dir_A`, `dir_C`, `dir_E`, `dir_G`, and `dir_I`,
+####         using a command with 8 characters (including spaces)
+
+rm -r d*
+
+#### Ex. 19: Create a directory called `nice_dir_name` and then use `echo`,
+####         redirecting its output to create a file called `nice_file_name`
+####         inside of `nice_dir_name` whose contents is the string
+####         `A string in a file!`
+
+mkdir nice_dir_name
+echo A string in a file! > nice_dir_name/nice_file_name
```

Comment:
This probably have an error, ja?  I forgot to tell everyone that the ! is a special character that must be put inside single quotes.  Oops! My bad.